packages/admin-next/dashboard/src/routes/products/product-detail/components/product-variant-section/use-variant-table-columns.tsx

```diff
@@ -37,6 +45,19 @@ const VariantActions = ({
     await mutateAsync()
   }
 
+  const handleInventoryItemNavigation = () => {
+    const itemId = variant.inventory_items![0].inventory.id
+    navigate(`/inventory/${itemId}`)
+  }
+
+  const handleInventoryKitNavigation = () => {
+    const itemIds = variant.inventory_items!.map((i) => i.inventory.id)
+    const params = { id: itemIds }
+    const query = new URLSearchParams(params).toString()
+
+    navigate(`/inventory?${query}`)
+  }
```

Comment:
minor thing, but could we update this functions to return the URL instead, and then use the `to` prop on the action items. That way it will actually be a link, which will be beter for screen readers and will allow us to prefetch the URL once we switch to RR7